Flutter/slider/how-to/custom-widget-on-flutterflow.md

```diff
@@ -0,0 +1,80 @@
+![Version](how-to-section-images/copy-version.png)
+3. Paste the copied dependency into the text editor, then click `Refresh` and `Save` it.
+
+>**Note**: The live version of [Syncfusion Flutter Sliders](https://pub.dev/packages/syncfusion_flutter_sliders) has been migrated to the latest version of Flutter SDK. To ensure compatibility, please check [FlutterFlow](https://app.flutterflow.io/dashboard)'s current Flutter version and obtain the corresponding version of [Syncfusion Flutter Sliders](https://pub.dev/packages/syncfusion_flutter_sliders) by referring to the [SDK compatibility](https://help.syncfusion.com/flutter/system-requirements#sdk-version-compatibility).
+
+![Dependency](how-to-section-images/dependency.png)
+
+>**Note**: If you are using an older version of a dependency instead of the latest one, remove the caret symbol (^) prefix in the version number after pasting the dependency. For example, change `^21.3.0` to `21.3.0`.
+>**Note**: Since [Syncfusion Flutter Sliders](https://pub.dev/packages/syncfusion_flutter_sliders) depends on the [Syncfusion Flutter Core](https://pub.dev/packages/syncfusion_flutter_core) package, make sure to add it as a dependency following the same steps mentioned above.
+
+### Import the package
+
+1. Navigate to the `Installing` tab on the [Syncfusion Flutter Sliders](https://pub.dev/packages/syncfusion_flutter_sliders) page. Under the `Import it` section, copy the package import statement.
+
+![Package](how-to-section-images/copy-package.png)
+
+2. Paste the copied import statement into the code editor and then `Save` it.
+
+![Import](how-to-section-images/import-package-flutterflow.png)
+
+### Add widget code snippet in code editor
+
+1. Navigate to the [Example](https://pub.dev/packages/syncfusion_flutter_sliders/example) tab in [Syncfusion Flutter Sliders](https://pub.dev/packages/syncfusion_flutter_sliders) and copy the widget specific codes.
+
+![Code](how-to-section-images/code-snippet.png)
+
+3. Paste the copied code snippet into the code editor, click `Format Code`, and `Save` it.
+
+![Code snippet](how-to-section-images/Adding-code-snippent.png)
+
+### Compiling the codes
+
+1. Click on the 'Compile Code' button located in the top right corner.
+2. If there are no errors, save the process. If errors are present, fix them and compile the code again. Once the code has been successfully compiled, save the process.
+
+![Compile code](how-to-section-images/compile-code.png)
+
+>**Note**: The compilation progress takes 2 to 3 minutes to complete.
+
+### Utilizing the custom widget
+
+1. Navigate to `Widget Palette` located in the left side navigation menu.
+2. Click on the `Componenets` tab.
+3. Under `Custom Code Widgets`, your custom widget will be there. Now, the custom widget can be added to your page by drag-and-drop.
```

Comment:
3. Your custom widget will be under `Custom Code Widgets`. Drag and drop the custom widget to your page.